rotkehlchen/accounting/structures/types.py

```diff
@@ -104,3 +104,55 @@ class HistoryEventSubType(SerializableEnumNameMixin):
 
     def serialize_or_none(self) -> Optional[str]:
         return self.serialize()
+
+
+class EventDirection(SerializableEnumNameMixin):
+    """Describes the direction of asset's movement (to the user, from the user or neutral)"""
+    IN = auto()
+    OUT = auto()
+    NEUTRAL = auto()
+
+
+class EventCategoryDetails(NamedTuple):
+    label: str
+    icon: str
+    direction: EventDirection
+    color: Optional[Literal['green', 'red']] = None
+
+
+class EventCategory(SerializableEnumNameMixin):
+    """
+    User friendly categories to classify combinations of event type and event subtype
+    """
+    GAS = auto()
+    SEND = auto()
+    RECEIVE = auto()
+    SWAP_OUT = auto()
+    SWAP_IN = auto()
+    APPROVAL = auto()
+    DEPOSIT = auto()
+    WITHDRAW = auto()
+    AIRDROP = auto()
+    BORROW = auto()
+    REPAY = auto()
+    DEPLOY = auto()
+    BRIDGE_IN = auto()
+    BRIDGE_OUT = auto()
+    GOVERNANCE = auto()
+    DONATE = auto()
+    RECEIVE_DONATION = auto()
+    RENEW = auto()
+    PLACE_ORDER = auto()
+    TRANSFER = auto()
+    CLAIM_REWARD = auto()
+    LIQUIDATE = auto()
+    INFORMATIONAL = auto()
+    CANCEL_ORDER = auto()
+    REFUND = auto()
+    FEE = auto()
+    MEV_REWARD = auto()
+    STAKING_REWARD = auto()
+    CREATE_BLOCK = auto()  # used only in frontend
```

Comment:
What do you mean by that?

```suggestion
    CREATE_BLOCK = auto()
```

Reply:
added a note because I was checking for missing stuff and then I found the combination using it but forgot that I had this comment. Should be removed

Reply:
Resolved with suggested change.